TypeScript.YAML-tmLanguage

```diff
@@ -26,11 +26,18 @@ repository:
       '1': { name: variable.ts }
     end: (?=$|[;,])
     patterns:
+    - include: '#ternary-operator'
     - include: '#type-annotation'
     - include: '#string'
     - include: '#comment'
     - include: '#expression'
 
+  ternary-operator:
```

Comment:
Is it safe just to end there? I know an expression can exist anywhere, but not any production can come after the ":". For example, is this now parsed as a code block (with a labeled statement) after the ":" instead of an expression:

```javascript
var x = true ? {a: 42} : {label: "use strict"};
```